tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php

```diff
@@ -275,7 +275,7 @@ public function addsMissingHtml5DocumentType()
     }
 
     /**
-     * @return string[][]
+     * @return array<string, array<int, string>>
```

Comment:
We need a `@psalm-return` here, or it will break some IDEs like PhpStorm.

Reply:
see caaba85